ci_scripts/install-holochain.sh

```diff
@@ -8,3 +8,5 @@ cargo install --force holochain \
 cargo install --force dna_util \
   --git https://github.com/holochain/holochain.git \
   --rev $REV
+cargo install --force lair-keystore \
```

Comment:
you can use --rev and pass in a tag. that'd prby be best since what you've got here will default to tip of develop or master (whichever is default) and eventually break when there's a next release, if not well coordinated

Reply:
done